Energy Saver/Pages/Statistics.cshtml.cs

```diff
@@ -101,15 +101,7 @@ public List<DataWithLabel> CreateDataForYearChart(int year)
 
                 foreach (Months month in Enum.GetValues(typeof(Months)))
                 {
-                    double? foundData;
-                    try
-                    {
-                        foundData = Taxes.SelectMany(x => x.Where(tax => tax.Year == year && tax.Month == month).Select(function)).First();
-                    }
-                    catch (InvalidOperationException)
-                    {
-                        foundData = 0;
-                    }
+                    double? foundData = Taxes.SelectMany(x => x.Where(tax => tax.Year == year && tax.Month == month).Select(function)).FirstOrDefault();
```

Comment:
Not sure if this change is needed. The chart looks odd now, has holes in it not all months in a year have taxes

Reply:
It does exactly the same that it used to do, because FirstOrDefault() will write 0 if no data is found

Reply:
Forgot that double? defaults to null, reverted the changes